src/Umbraco.Web/Runtime/WebInitialComposer.cs

```diff
@@ -31,7 +31,6 @@ public override void Compose(Composition composition)
 
             composition.Register<UmbracoInjectedModule>();
 
-            composition.ComposeWebMappingProfiles();
```

Comment:
I don't know what the implications of this will be since it has been deleted, but are we registering all the previous components somewhere else now (or future task)?

Reply:
I moved some of the registrations to the `ComposeCoreMappingProfiles`, and the rest is added in `BackOfficeComposer`